File: Algoritmos Traduzidos/Ruby/Algoritmos_Tiago/Alg_Volume_Paralelipipedo.rb
#Elabore um algoritmo que peça o valor da altura, do lado maior e menor e imprima o volume do paralelipipedo.
puts "Introduza o lado maior, lado menor e altura por esta ordem";
lmaior = gets
lmenor = gets
altura = gets
volume = lmaior.to_f * lmenor.to_f * altura.to_f
puts "O volume e: #{volume}"

File: Algoritmos Traduzidos/Perl/Algoritmos_Tiago/Alg_Volume_Paralelipipedo.pl
#Elabore um algoritmo que peça o valor da altura, do lado maior e menor e imprima o volume do paralelipipedo.
print "Introduza o lado maior, lado menor e altura por esta ordem\n";
$lmaior = <STDIN>;
$lmenor = <STDIN>;
$altura = <STDIN>;
$volume = $lmaior * $lmenor * $altura;
print "O volume e: $volume\n";